charts/kafka-connect/templates/kafka-connect.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaConnect
+metadata:
+  name: kafka
+  namespace: {{ .Release.Namespace }}
```

Comment:
Does helm not manage this automatically?

Reply:
will test again without

Reply:
it works without indeed